src/main/java/net/dries007/tfc/client/ClientEvents.java

```diff
@@ -206,7 +216,6 @@ else if (mc.world.provider.getDimension() == 0)
                     data.getValidTrees().forEach(t -> list.add(String.format("%s %s (%.1f)", WHITE, t.getRegistryName(), t.getDominance())));
 
                     list.add(GRAY + "Sea level offset: " + WHITE + data.getSeaLevelOffset(x, z));
-                    list.add(GRAY + "Spawn Protection: " + WHITE + data.getSpawnProtection());
```

Comment:
Why? This is the only available location for this data atm.

Reply:
Crud! I forgot to revert that! I removed at the earlier stages of development as we discussed that it would not be required after i moved mobs underground, but i decided to revert spawn protection since aggressive animals (predators) would still spawn at a player's base regardless of light sources, hence making this system much more needed